packages/unit-testing/stubs/workbook/Aspect.js

```diff
@@ -0,0 +1,32 @@
+define(['./CalculatedMeasure', './DataMeasure'], function (CalculatedMeasure, DataMeasure) {
```

Comment:
This workbook "subObject" is not appearing in documentation. I think we should leave it as it is, but leaving the comment to keep track of it.

Reply:
Same as with Category and Chart, there's createAspect function in N/workbook